### PR TITLE
Redirect and update several spell effects

### PR DESCRIPTION
**Pull request by pedrogrullada**
- Trim down and add "Granted by [...]" paragraphs to several effects
- Grant Thundering Roar with Thundering Dominance effect, and add scaling to its damage.
- Use origin signature in consolidated Draw Ire spell effects
- Localise and trim down Organsight spell effect rule elements
- Consolidate Clawsong spell effects
- Consolidate Community Repair spell effects
- Consolidate Dread Ambience spell effects
- Consolidate Touch of the Void spell effects
  - Refresh copies of the spells in Bestiaries
- Consolidate Warding Aggression spell effects
- Redirect several consolidated spell effects

### CODE REVIEW

File: packs/blood-lords-bestiary/ghoul-antipaladin.json [hunk 1]
```diff
@@ -39,14 +39,9 @@
             "type": "spellcastingEntry"
         },
         {
-            "_id": "kB1mIppy8e01CEe2",
-            "flags": {
-                "core": {
-                    "sourceId": "Compendium.pf2e.spells-srd.Item.jFmWSIpJGGebim6y"
-                }
-            },
+            "_id": "bEdDxGUh3lcv55Pd",
```
Comment (simonward) [inline packs/blood-lords-bestiary/ghoul-antipaladin.json:42]: All the sourceIds were stripped. How are you refreshing?

Comment (pedrogrullada) [reply packs/blood-lords-bestiary/ghoul-antipaladin.json:42]: Just dragged the spell again. The refresh button wasn't working for me, which I was going to report tomorrow after I made sure it wasn't already known, or just user error.

Comment (simonward) [reply packs/blood-lords-bestiary/ghoul-antipaladin.json:42]: Yeah, I found the same with the refresh button. https://github.com/foundryvtt/pf2e/issues/16512

If dragging fresh copies is losing the sourceId that may be a separate issue.

**Comment by TikaelSol** [comment]
I'm hesitant to blanket replace spells on older NPCs.

**Comment by pedrogrullada** [comment]
> I'm hesitant to blanket replace spells on older NPCs.
Could just remove the old links manually then

File: packs/spell-effects/spell-effect-draw-ire.json [hunk 2]
```diff
@@ -36,20 +36,16 @@
...
                 "predicate": [
                     {
-                        "not": "target:drawIre"
+                        "not": "target:signature:{item|origin.signature}"
                     }
                 ],
                 "selector": "attack",
                 "type": "status",
-                "value": "{item|flags.pf2e.rulesSelections.degreeOfSuccess}"
+                "value": "@item.flags.pf2e.rulesSelections.degreeOfSuccess}"
```
Comment (simonward) [inline packs/spell-effects/spell-effect-draw-ire.json:48]: Looks like an extra closing curly.

**Pull request merged**